services/api_service/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM python:3.9.6-alpine
+
+WORKDIR /usr/src/api_service
+
+ENV PYTHONDONTWRITEBYTECODE 1
+ENV PYTHONUNBUFFERED 1
+
+RUN adduser -D user
```

Comment:
why do you need this?

Reply:
for security purpose. 
By default, Docker containers run as root.
That root user is the same root user of the host machine, with UID 0. This fact can enable hackers to perform various types of attacks on your app if they get hold of your vulnerable container: they can copy sensitive files from the host to the container and access them.

For that reason, it is recommended to create user. Even in terminal output you will receive a couple of messages saying this.